tests/unit/nulls.c

```diff
@@ -146,6 +146,23 @@ int main(int argc, char *argv[])
   CHECKED_QUERY(querybuf);
 
 #ifdef TEST_PREPARED_STATEMENTS
+  strcpy(querybuf, "SELECT NULL");
+  sth = drizzle_stmt_prepare(con, querybuf, strlen(querybuf), &driz_ret);
+
+  ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, preparing \"%s\"",
+             drizzle_strerror(driz_ret), drizzle_error(con), querybuf);
+
+  driz_ret = drizzle_stmt_execute(sth);
+  ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, executing \"%s\"",
+    drizzle_strerror(driz_ret), drizzle_error(con), querybuf);
```

Comment:
Should be the same as in line [\#152](https://github.com/sociomantic-tsunami/libdrizzle-redux/pull/200/files/a90efb587f9c1ce5af55af7637278b2cfaddc95b#diff-02a60f8dc8105e273191327adaaf11b6R152)
@chenzhuoyu could you fix that?